### PR TITLE
Fixed broken link to slideshow

### PR DESCRIPTION
**Pull request by jt15s**
Fixed a broken link to the Haiku slideshow which linked to a non-existent page.

### CODE REVIEW

File: content/about/faq.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ into the Haiku code base. The rest is either homebuilt code or derivatives of ex
 is mostly compatible with the BeOS API. In other words, many applications that were initially developed for BeOS work flawlessly on Haiku.
 
 ### I've never seen Haiku. What does it look like?
-Check out our [Haiku Slideshow](/slideshows/) and the [Haiku Movies](/about/movies) pages.
+Check out our [Haiku Slideshow](/slideshows/haiku-1/) and the [Haiku Movies](/about/movies) pages.
```
Comment (nielx) [inline content/about/faq.md:110]: The [about page](https://github.com/haiku/website/blob/master/content/about/_index.md) links to /slideshows/haiku-tour which rewrites to this particular URL. I do not know which one is preferred.

Comment (pulkomandy) [reply content/about/faq.md:110]: haiku-tour makes more sense. haiku-1 is just the first page of the "tour".

Comment (jt15s) [reply content/about/faq.md:110]: Oh ok. I'll fix that.

Comment (jt15s) [reply content/about/faq.md]: Done.

**Pull request merged**